flink-contrib/flink-storm-examples/src/main/java/org/apache/flink/storm/util/BoltFileSink.java

```diff
@@ -40,16 +43,17 @@ public BoltFileSink(final String path) {
 
 	public BoltFileSink(final String path, final OutputFormatter formatter) {
 		super(formatter);
-		this.path = path;
+		this.path = new Path(path);
 	}
 
 	@SuppressWarnings("rawtypes")
 	@Override
 	public void prepareSimple(final Map stormConf, final TopologyContext context) {
 		try {
-			this.writer = new BufferedWriter(new FileWriter(this.path));
+			FSDataOutputStream outputStream = FileSystem.getLocalFileSystem().create(path, false);
+			this.writer = new BufferedWriter(new OutputStreamWriter(outputStream));
 		} catch (final IOException e) {
```

Comment:
I had problems if the path contained a file scheme without the file system abstraction.